src/V3Undriven.cpp

```diff
@@ -381,7 +413,40 @@ class UndrivenVisitor final : public VNVisitor {
                     UINFO(9, " Full bus.  Entryp=" << cvtToHex(entryp) << endl);
                     warnAlwCombOrder(nodep);
                 }
-                entryp->drivenWhole();
+                if (entryp->isDrivenWhole() &&
+		    !m_inBBox &&
+		    !VN_IS(nodep, VarXRef) &&
+		    !nodep->underGenerate() &&
```

Comment:
Why is this suppressed inside generates?

Reply:
The check was misfiring quite badly inside an always_comb within a generate, namely in the test for t_split_var_0.v

V3Undriven seems to set `drivenWhole()` on multi-bit vars that are only partially assigned within a generate. (I think it treats them as individual vars?)

Also V3Undriven seems to treat a var assigned within an always_comb of a generate loop as being within multiple always_comb blocks. Which makes sense, as multiple always_comb blocks are being 'generated'.  But in conjunction with the other behavior described above, the end result was that the check misfired

Reply:
That makes sense in retrospect.  A hack that might work is that the duplicates will have the same source fileline(), so you could consider duplicate filelines as non-conflicting.